tests/logging/Makefile

```diff
@@ -31,3 +31,6 @@ ci-master: resolve validate build build-image push-image
 .PHONY: ci-release
 ci-release: resolve validate build build-image push-image
 
+#.PHONY: path-to-referenced-charts
```

Comment:
why is this commented out.

Reply:
There are some inconsistency between directory name and key inside chart values.yaml, all details you can find in this [comment](https://github.com/kyma-project/test-infra/issues/588#issuecomment-470487256). 
We decided create new issue for teams to fix inconsistency (issues are already in concrete team backlog's), until all fixes we commented this Makefile target to not block the feature.